dist/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+<script defer src="main.bundle.js"></script></head>
+<body>
+  <div class="container">
+    <div class="todolist">
+      <section class="title">
+        <h2>Today's To Do</h2>
+        <p class="img"></p>
+      </section>
+      <hr>
+      <div class="form">
+        <input type="text" class="inputF" placeholder="Add to your list...">
+        <p class="enter-img"></p>
```

Comment:
```suggestion
        <img class="enter-img" src="#" />
```

I think it would be great to use the `<img>` tag here instead of the `<p>` tag

Reply:
By using `img` tag the linter error is rising.